scripts/ci/run_ci.sh

```diff
@@ -24,6 +24,9 @@ SANITYCHECK_OPTIONS=" --inline-logs -N"
 SANITYCHECK_OPTIONS_RETRY="${SANITYCHECK_OPTIONS} --only-failed --outdir=out-2nd-pass"
 SANITYCHECK_OPTIONS_RETRY_2="${SANITYCHECK_OPTIONS} --only-failed --outdir=out-3nd-pass"
 export BSIM_OUT_PATH="${BSIM_OUT_PATH:-/opt/bsim/}"
+if [ ! -d ${BSIM_OUT_PATH} ]; then
```

Comment:
please quote it so it does not break if there is spaces in the path
`if [ ! -d "${BSIM_OUT_PATH}" ]; then`